pylint/checkers/base_checker.py

```diff
@@ -201,3 +202,32 @@ class BaseTokenChecker(BaseChecker):
     def process_tokens(self, tokens):
         """Should be overridden by subclasses."""
         raise NotImplementedError()
+
+
+class _BasicChecker(BaseChecker):
```

Comment:
As we're likely going to squash merge this PR it would be good to do this change and associated changes in `pylint/checkers/base.py` in a separate PR. I can this for you if you want?

Reply:
reverted